testsuite/tests/ui/test_unique_invoice_id.py

```diff
@@ -0,0 +1,65 @@
+"""
+rewrite of /spec/ui_specs/billing/unique_invoice_id_spec.rb
+For testing this issue, we don't need to use UI
+"""
+from datetime import date, timedelta
+import pytest
+from threescale_api.resources import InvoiceState, Account, ApplicationPlan
+
+pytestmark = pytest.mark.issue("https://issues.jboss.org/browse/THREESCALE-1203")
+
+NACCOUNTS = 10
+NEXT_DAY = (date.today() + timedelta(days=1)).isoformat()
+
+
+@pytest.fixture(scope="module")
+def paid_app_plan(service, custom_app_plan) -> ApplicationPlan:
+    """Application plan on default service with some setup fee"""
+    return custom_app_plan(dict(name="paidtestplan", setup_fee="100"), service=service)
```

Comment:
Please use `rawobj.ApplicationPlan`, it creates the dict for you. It is not the only instance where rawobj.* could be used so please change all of them.

Reply:
Alex we talked about this!

Reply:
Fixed and also added one field for ApplicationPlan rawobj